source/deps.deno.ts

```diff
@@ -0,0 +1,10 @@
+export { type Context } from "https://lib.deno.dev/x/grammy@v1/mod.ts";
+export type {
+  Message,
+  MessageEntity,
+} from "https://lib.deno.dev/x/grammy@v1/types.ts";
+export {
+  html,
+  markdown,
+  markdownv2,
+} from "https://esm.sh/telegram-format@2.1.0";
```

Comment:
What would you suggest?
1. Import from esm.sh (works fine)
2. Port telegram-format first? (worth the work?)
3. Do not import at all. Do it natively (maybe, use the url code from telegram-format)

Reply:
Personally I do like using telegram-format so I don't need to think about the actual formatting. So it should be also available via Deno especially as its without dependencies. But it also works fine from esm.sh because of this so I'm not sure its worth the effort to do it manually.

As this repo only needs the urls it might be simpler to integrate it with the suffix methods.

Reply:
I agree that it is simpler to just integrate them here, as we can remove a dependency as well. I'll update.

Reply:
I forgot the escaping part which also needs to happens there… (good thing I normally use telegram-format for that and dont need to remember stuff like that).

Maybe the escaping is buggy or whatever and a dedicated library which can have its own bugfixes might be a smart thing then…

So the question returns… port or esm.sh?
Does it change things with deno2node to have dependencies like that?

Reply:
While importing from esm.sh works, I have a feeling that porting the library to Deno would be better. There won't be much changes in deno2node setup either way. But I'm not sure if it worth the work, even tho its not that much.

So, I guess I'll leave you and @KnorpelSenf to deciding what would be better. Sorry about the late reply.

Reply:
I always prefer native Deno modules if someone is willing to spend the time

Reply:
Thanks. I got some time to spend now.

Reply:
Did you use any of your time on this?

Reply:
Guess one open thing is this: https://github.com/EdJoPaTo/telegram-format/pull/36

Personally I should invest time to get this Deno first approach right and roll it out to multiple of my repos… Sadly Deno is a bit mixed in recent times and stuff like a type aware linter is missing so my priority isnt that high on that currently :grimacing: